wave_vision/include/wave/vision/matcher/descriptor_matcher.hpp

```diff
@@ -89,7 +89,7 @@ class DescriptorMatcher {
       const cv::Mat &descriptors_2,
       const std::vector<cv::KeyPoint> &keypoints_1,
       const std::vector<cv::KeyPoint> &keypoints_2,
-      cv::InputArray mask = cv::noArray()) const = 0;
+      cv::InputArray mask) const = 0;
```

Comment:
Any suggestions? I'm not sure what the best way to do this is. The default should be no mask. 

I want to say these classes will not be used in a polymorphic manner...but that's always a dangerous assumption. I'm going to add back the default parameter in the base class for now.